app/views/hyrax/dashboard/sidebar/_repository_content.html.erb

```diff
@@ -1,22 +1,22 @@
 <%# override from bulkrax to make bulkrax a feature flipper %>
 <li class="h5"><%= t('hyrax.admin.sidebar.repository_objects') %></li>
 
-<% if current_user.can?(:manage, Collection) || Flipflop.enabled?(:show_collections_link) %>
```

Comment:
I have some slight concern about moving this link below works for all cases.  I think it does make sense for the works link to stay constant at the top of the list when other links are appearing/disappearing based upon role, but if the feature isn't disabled then I think this changes the upstream behavior and I'm not sure if there is a strong enough case to change that behavior..

Reply:
It was mostly just because otherwise we would need to introduce another loop into the view and considering it is doing the exact same thing, moving the links seemed the better option